examples/jsm/loaders/VTKLoader.js

```diff
@@ -62,6 +62,8 @@ VTKLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 			var result;
 
+			var patWord = /^[^0-9.\s-]+/;
```

Comment:
It seems this line is different that in the `js` file?

Reply:
Ho yes, it should be equivalent. I will just fix it, otherwise we can run the conversion script if that's easier for you.

Reply:
It's good when both files are in sync in a PR.